core/mcp-client/CHANGELOG.md

```
@@ -0,0 +1,1009 @@
+# [3.10.0](https://github.com/eggjs/tegg/compare/v3.9.0...v3.10.0) (2023-06-28)
+
+**Note:** Version bump only for package @eggjs/tegg-orm-decorator
+
+
+
+
+
+# [3.8.0](https://github.com/eggjs/tegg/compare/v3.7.0...v3.8.0) (2023-05-30)
+
+
+### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)